src/components/Selectors/selectors.module.scss

```diff
@@ -0,0 +1,233 @@
+.selector {
```

Comment:
can we split this into two files radio and checkbox. also can we use variables instead of the hard coded px values